tests/e2e/specs/stories-editor/page.js

```diff
@@ -218,4 +248,102 @@ describe( 'Story Page', () => {
 		const previewPage = await openPreviewPage( editorPage, 'amp-story' );
 		await previewPage.waitForXPath( '//amp-video[contains(@aria-label, "Hello World")]' );
 	} );
+
+	it( 'should pass global story settings to a new story', async () => {
+		const documentPanelSelector = 'button.edit-post-sidebar__panel-tab[data-label="Document"]';
+		const storySettingsPanelSelector = '.amp-story-settings-panel';
+		const storySettingsAdvanceAfterSelector = '.components-panel__body.amp-story-settings-panel .components-select-control__input:nth-of-type(1)';
```

Comment:
You could add a custom class to the select menu here, to avoid kind of expensive selector.